### PR TITLE
frame rebalance

### LINKED ISSUE #1123: Machine Frame material return balancing
Description of issue: Deconstructing machine frames gives you 5 metal instead of 8.

Difference between expected and actual behavior: It should give you 8 but honestly I hate how machine frames are an uneven number like 8 as it makes machine construction even harder when the game has so few materials to utilize roundstart without greytiding.

Steps to reproduce: Wrench a machine frame.

### PR DESCRIPTION
**Pull request by Evankhell561**
fix:  #1123 
:cl:
fix: The materials that the frames of various machines / structures drop do not correspond to the amount of materials needed to create them.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/ai_core.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 			if(isWelder(P))
 				if(P.use_tool(user, src, WORKTIME_NORMAL, QUALITY_WELDING, FAILCHANCE_NORMAL))
 					to_chat(user, "<span class='notice'>You deconstruct the frame.</span>")
-					new /obj/item/stack/material/plasteel( loc, 4)
```
Comment (NanakoAC) [inline code/game/machinery/computer/ai_core.dm:36]: i'm assuming you checked the craft menu to see what these should cost?

**Review by NanakoAC — APPROVED** [review]

File: code/game/machinery/computer/ai_core.dm (hunk 1 above)
Comment (Evankhell561) [reply code/game/machinery/computer/ai_core.dm:36]: image

File: code/game/machinery/constructable_frame.dm [hunk 2]
```diff
@@ -45,7 +45,7 @@
 					if(isWrench(P))
 						playsound(src.loc, 'sound/items/Ratchet.ogg', 75, 1)
 						to_chat(user, "<span class='notice'>You dismantle the frame</span>")
-						new /obj/item/stack/material/steel(src.loc, 5)
+						new /obj/item/stack/material/steel(src.loc, 8)
```
Comment (Evankhell561) [inline code/game/machinery/constructable_frame.dm:48]: image

File: code/game/machinery/computer/buildandrepair.dm [hunk 3]
```diff
@@ -23,7 +23,7 @@
 			if(isWelder(P))
 				if(P.use_tool(user, src, WORKTIME_NORMAL, QUALITY_WELDING, FAILCHANCE_NORMAL))
 					to_chat(user, "<span class='notice'>You deconstruct the frame.</span>")
-					new /obj/item/stack/material/steel( src.loc, 5 )
+					new /obj/item/stack/material/steel( src.loc, 8 )
```
Comment (Evankhell561) [inline code/game/machinery/computer/buildandrepair.dm:26]: image

**Pull request merged**